src/devices/src/virtio/vsock/unix/muxer.rs

```diff
@@ -1339,4 +1339,33 @@ mod tests {
         // Check that fwd_cnt is 0 - "OK ..." was not accounted for.
         assert_eq!(conn.fwd_cnt().0, 0);
     }
+
+    #[test]
+    fn test_regression_rxq_pop() {
```

Comment:
Same comment as for the other test. It would be useful to also have a link to the GitHub issue that this regression test is addressing.

Reply:
Link added